docs/conf.py

```diff
@@ -174,6 +172,9 @@
 napoleon_google_docstring = False
 extensions += ['sphinx_astropy.ext.edit_on_github', 'sphinx.ext.doctest', 'sphinx.ext.githubpages']
 
+# Remove numpydoc
```

Comment:
Can you put this directly above the ` extensions.append('sphinx.ext.napoleon')` line?

Also the napoleon config seems to be duplicated?  Can you remove the one not under the "swap to napoleon" comment?